docs/features/features-walk-through/metadata_only.md

```diff
@@ -0,0 +1,10 @@
+# Metadata-only records
+
+
+A metadata-only record is a record that contains only descriptive metadata, without any associated files or content. These types of records can be useful in cases where you want to provide metadata for a resource that does not have a corresponding digital object or content file.
+
+The site administrator can choose to enable and disable the creation of metadata-only records.
```

Comment:
Maybe mentioning "how" to do so, i.e. `METADATA_ONLY_ENABLED=True`, or pointing (hyperlink) to the place in the docs where this configuration variable is documented.